app/javascript/controllers/dropzone_controller.js

```diff
@@ -61,7 +64,6 @@ export default class extends Controller {
         if (this.basePath) path = `${this.basePath}/${path}`
         data.append(`content[paths][${i}]`, path)
       }
-      data.append('content[completed]', this.dropzone.getActiveFiles().length === files.length)
```

Comment:
This mechanism was unreliable because the upload of files was asynchronous, which meant that files might still be active even though they passed through this callback.